src/gluonts/transform/feature.py

```diff
@@ -57,21 +63,58 @@ def __init__(
         target_field: str,
         output_field: str,
         dummy_value: float = 0.0,
+        imputation_method: str = "standard",
         convert_nans: bool = True,
         dtype: DType = np.float32,
     ) -> None:
         self.dummy_value = dummy_value
         self.target_field = target_field
         self.output_field = output_field
+        self.imputation_method = imputation_method
         self.convert_nans = convert_nans
         self.dtype = dtype
 
+    def _dummy_impute_missing(self, value: np.ndarray) -> np.ndarray:
+        nan_indices = np.where(np.isnan(value))
+        value[nan_indices] = self.dummy_value
+        return value
+
+    def _mean_impute_missing(self, value: np.ndarray) -> np.ndarray:
+        nan_indices = np.where(np.isnan(value))
+        value[nan_indices] = np.nanmean(value)
+        return value
+
+    def _median_impute_missing(self, value: np.ndarray) -> np.ndarray:
+        nan_indices = np.where(np.isnan(value))
+        value[nan_indices] = np.nanmedian(value)
```

Comment:
I think that both mean and median, implemented this way, will “leak” future observations into past missing values. The running mean and median should be used in order to avoid that, but I’m not sure how efficiently these can be implemented?